drivers/hmis/app/graphql/types/hmis_schema/disability_group.rb

```diff
@@ -16,6 +16,8 @@ class HmisSchema::DisabilityGroup < Types::BaseObject
     field :information_date, GraphQL::Types::ISO8601Date, null: false
     field :data_collection_stage, HmisSchema::Enums::Hud::DataCollectionStage, null: false
 
+    field :disabling_condition, HmisSchema::Enums::Hud::NoYesReasonsForMissingData, 'Current disabling conditionn on the linked Enrollment. It may not matching up with the disabilities specified in this group.', null: false
```

Comment:
```suggestion
    field :disabling_condition, HmisSchema::Enums::Hud::NoYesReasonsForMissingData, 'Current disabling conditionn on the linked Enrollment. It may not match up with the disabilities specified in this group.', null: false
```